src/client/customers.rs

```diff
@@ -152,6 +158,8 @@ pub struct Customer {
     pub name: String,
     /// A valid email for the customer, to be used for notifications.
     pub email: String,
+    /// Additional email addresses for this customer.
+    pub additional_emails: Vec<String>,
```

Comment:
In orb doc it says required field in response body - need to confirm always returned in response payload even if empty list

Reply:
covered by test